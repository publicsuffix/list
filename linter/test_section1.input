// test:
// - no section at all

example.com
