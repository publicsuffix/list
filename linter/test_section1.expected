4: error: Rule outside of section: 'example.com'
4: warning: No ICANN section found
4: warning: No PRIVATE section found
